.ls-lint.yml

```diff
@@ -17,3 +17,4 @@ ignore:
   - node_modules
   - SECURITY.md
   - test/e2e/node_modules
+  - lib/types/renovate_pep440.d.ts
```

Comment:
Does it need to be this file name?

Reply:
No, the name isn't important.

Reply:
We should move the types to https://github.com/renovatebot/pep440 now, before we merge this PR.

Reply:
I have created a PR for the same [link](https://github.com/renovatebot/pep440/pull/113)